setup.py

```diff
@@ -99,13 +98,14 @@ def read(*rnames):
     tests_require=TESTS_REQUIRE,
     install_requires=[
         'setuptools',
-        'zope.interface>=4.1.0',
+        'zope.interface >= 4.1.0',
         'zope.event',
+        'zope.hookable >= 4.2.0',
     ],
     include_package_data=True,
     zip_safe=False,
     extras_require={
-        'hook': HOOK_REQUIRES,
+        'hook': HOOK_REQUIRES, # BWC
```

Comment:
FWIW, we normally use `BBB` for that purpose.  Also, linters are likely to barf without at least two spaces before the `#`.